Core/clim-core/standard-presentations.lisp

```diff
@@ -110,6 +91,14 @@
 (define-presentation-method presentation-typep (object (type symbol))
   (symbolp object))
 
+(define-presentation-method presentation-typep (object (type (eql t)))
+  (declare (ignore object))
+  t)
+
+(define-presentation-method presentation-typep (object (type (eql nil)))
+  (declare (ignore object))
+  nil)
+
```

Comment:
Could you add a test for these?

Reply:
```
(test presentations.typep.1
  (define-presentation-type foo ())
  (is (null (presentation-typep 3 'foo))))
```

added (and this test indeed fails on current master)